src/cmd/flux-python

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# This minimal wrapper exists to help handle systems where env -S is
+# unavailable and scripts wish to use `flux python`. Using a shebang like this
+# for flux python scripts allows them to rely only on posix to start:
+# #!/usr/bin/env flux-python
+
+/usr/bin/env flux python "$@"
```

Comment:
Probably want to avoid an extra process:

```suggestion
exec /usr/bin/env flux python "$@"
```

Reply:
while we're at it, is the `/usr/bin/env`  superfluous in a wrapper script?  (`/bin/sh` will already find the first `flux` in `PATH`) I could be missing some subtlety though.

Reply:
True, it is superfluous.  Fixing.